src/logic/calculate.js

```diff
@@ -0,0 +1,41 @@
+import operate from './operate';
+
+const calculate = (data, btn) => {
+  let { total, next, operation } = data;
+
+  switch (btn) {
```

Comment:
### Heads-Up

we need to add more cases to pass the next milestone.

1. if the `buttonName` = `A/C`, we should return `0`
2. if the `buttonName`  = `.` we should return `0.`
3. if a number is divided by 0 we should return an error message like `division errror`